cabal.project

```diff
@@ -47,23 +47,21 @@ if os (windows)
 
 if impl (ghc >= 9.12)
   allow-newer:
-    -- Bounds will be relaxed in next release
-    , network-mux:base
-    , ouroboros-network-api:base
-    , ouroboros-network-framework:base
-    , ouroboros-network-protocols:base
-    , ouroboros-network:base
-
     -- https://github.com/kapralVV/Unique/issues/11
     , Unique:hashable
 
-source-repository-package
-   type: git
-   location: https://github.com/IntersectMBO/ouroboros-network
-   tag: 3e8d3b4b8c87ead794876c62d7fe25f32efb5142
-   --sha256: 08fpkx3iagj83nn413h9a865zjcj3lrf7017a756qd2wg2jg3amq
-   subdir:
-     ouroboros-network-api
+allow-newer:
+    -- https://github.com/phadej/vec/issues/121
+    , ral:QuickCheck
+    , fin:QuickCheck
+    , bin:QuickCheck
+
+    -- https://github.com/well-typed/quickcheck-lockstep/issues/38
+    , quickcheck-lockstep:QuickCheck
+    , quickckech-lockstep:quickcheck-dynamic
+
+    -- https://github.com/input-output-hk/fs-sim/pull/105
+    , fs-sim:QuickCheck
```

Comment:
If you update hackage to `2025-07-22T09:13:54Z` then you don't need this allow-newer (see https://hackage.haskell.org/package/fs-sim-0.4.0.0/revisions/)